octopusdeploy_framework/schemas/tenant.go

```diff
@@ -108,7 +108,7 @@ func (t TenantSchema) GetDatasourceSchema() datasourceSchema.Schema {
 						"id":          GetIdDatasourceSchema(true),
 						"name":        GetReadonlyNameDatasourceSchema(),
 						"space_id":    GetSpaceIdDatasourceSchema("tenant", true),
-						"tenant_tags": datasourceSchema.ListAttribute{
+						"tenant_tags": datasourceSchema.SetAttribute{
```

Comment:
Do we need a migration for this or will existing configurations with lists already in state just handle swapping to set fine?

Reply:
Yeah, it does. Having said that, the existing configuration doesn't work at all, anyway. If they did use the workaround to sort the tags alphabetically on the resource, it should just work. The state files don't look any different to what it was before.